ts/features/fci/store/reducers/fciSignatureRequest.ts

```diff
@@ -80,6 +80,9 @@ export const fciDocumentsWithUrlSelector = (
     )
   );
 
+/**
+ * @deprecated Use the new {@link fciEnvironmentSelector} instead.
+ */
 export const fciIssuerEnvironmentSelector = createSelector(
```

Comment:
If it is deprecated, can we remove it and replace its only usage in `ts/features/fci/saga/networking/handleGetQtspMetadata.ts` with the new one?

Reply:
We are maintaining this function for backward compatibility due to its generation of a distinct set of env enumerations for the `getQtspClausesMetadata` endpoint. Once it is replaced in the backend, we will remove this deprecated function as well.